vars/feedhenrySdksContainerComponentUpdate.groovy

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/groovy
+
+def call(Map parameters) {
+    String componentName = parameters.componentName
+    String componentVersion = parameters.componentVersion
+    String componentBuild = parameters.componentBuild
+    String sourceBranch = parameters.sourceBranch ?: ''
+    String targetBranch = parameters.targetBranch ?: ''
```

Comment:
Is `''` an acceptable fallback value for sourceBranch and targetBranch?

Reply:
Yes, this is the same as all the other ones (fhcap, openshift-templates etc..), and really we should refactor these. If the PR option is set these branch options should be blank since the job will determine the correct target and source branch from the PR.